eng/Directory.Build.Data.props

```diff
@@ -98,6 +98,8 @@
     <RequiredTargetFrameworks>netcoreapp2.1;net5.0</RequiredTargetFrameworks>
     <!-- Also test net461 on Windows; it's listed first so that coverage reports are for netcoreapp2.1 (the "primary"). -->
     <RequiredTargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net461;netcoreapp2.1;net5.0</RequiredTargetFrameworks>
+    <!-- Include .NET core 3.1 to target the perf projects as it is the current LTS version with the furthest End of Support time line. -->
+    <RequiredTargetFrameworks Condition="'$(IsPerfProject)' == 'true'">$(RequiredTargetFrameworks);netcoreapp3.1</RequiredTargetFrameworks>
```

Comment:
This is interesting, why not 2.1? It's the oldest supported LTS.

Reply:
2.1 is already in the list.  We also wanted to add the latest LTS .NET core framework.

Reply:
It's interesting that we are testing 2.1 but benchmarking 3.1. I think we need to pick one. cc @mikeharder

Reply:
Wasn't this what the conversation that Mike started in Teams was the other day?   There was consensus around 3.1, including yours.  😄

Reply:
Yep. I didn't realize we unit test 2.1. Somehow I thought it was 3.1.

Reply:
Non-blocking. We can continue the conversation and adjust later.